multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -69,23 +58,18 @@ type MemberClusterAnnounceReconciler struct {
 	Scheme *runtime.Scheme
 
 	mapLock      sync.RWMutex
-	memberStatus map[common.ClusterID]*multiclusterv1alpha1.ClusterStatus
-	timerData    map[common.ClusterID]*timerData
+	memberStatus map[common.ClusterID]*memberData
```

Comment:
Suggest to rename it memberStatusMap or memberStatuses

Reply:
Done